examples/touch.c

```diff
@@ -120,14 +273,22 @@ int main(int argc, char *argv[]) {
 		exit(EXIT_FAILURE);
 	}
 
-	if (!wlr_backend_start(compositor.backend)) {
+	const char *socket = wl_display_add_socket_auto(display);
+	if (!socket) {
+		wlr_log_errno(L_ERROR, "Unable to open wayland socket");
+		wlr_backend_destroy(wlr);
+		exit(1);
+	}
+	setenv("_WAYLAND_DISPLAY", socket, true);
```

Comment:
I don't think creating a socket is required at all.

Reply:
Yeah you're right that shouldn't be required